### ISSUE TITLE
Analysis does not complete until timeout

### ISSUE THREAD

**Issue by xulifan** [cuckoosandbox/cuckoo#788]
Hi, I setup Cuckoo and a VirtualBox machine as instructed in the Cuckoo website. 
However, After I submit a file to Cuckoo, it keeps saying "analysis not completed yet (status=2)", then timeout. Here is the log:

2016-02-27 19:44:00,250 [lib.cuckoo.core.resultserver] DEBUG: ResultServer running on 192.168.56.1:2042.
2016-02-27 19:44:00,253 [lib.cuckoo.core.scheduler] INFO: Using "virtualbox" as machine manager
2016-02-27 19:44:00,478 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:00,857 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status running
2016-02-27 19:44:00,891 [modules.machinery.virtualbox] DEBUG: Stopping vm cuckoo1
2016-02-27 19:44:00,892 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:01,236 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status running
2016-02-27 19:44:04,901 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:05,362 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status poweroff
2016-02-27 19:44:05,435 [lib.cuckoo.core.scheduler] INFO: Loaded 1 machine/s
2016-02-27 19:44:05,484 [lib.cuckoo.core.scheduler] INFO: Unable to determine the network interface for VM with name cuckoo1, Cuckoo will not be able to give it full internet access or route it through a VPN! Please define a default network interface for the machinery or define a network interface for each VM.
2016-02-27 19:44:05,484 [lib.cuckoo.core.scheduler] INFO: Waiting for analysis tasks.
2016-02-27 19:44:28,875 [lib.cuckoo.core.scheduler] DEBUG: Processing task #1
2016-02-27 19:44:28,879 [lib.cuckoo.core.scheduler] INFO: Starting analysis of FILE "cloudmusic.exe" (task #1, options "")
2016-02-27 19:44:30,106 [lib.cuckoo.core.scheduler] INFO: Task #1: acquired machine cuckoo1 (label=cuckoo1)
2016-02-27 19:44:30,108 [modules.auxiliary.sniffer] ERROR: Network interface not defined, network capture aborted
2016-02-27 19:44:30,108 [lib.cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2016-02-27 19:44:30,174 [modules.machinery.virtualbox] DEBUG: Starting vm cuckoo1
2016-02-27 19:44:30,174 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:30,267 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status poweroff
2016-02-27 19:44:30,468 [modules.machinery.virtualbox] DEBUG: Using current snapshot for virtual machine cuckoo1
2016-02-27 19:44:31,326 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:31,443 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status saved
2016-02-27 19:44:48,236 [modules.machinery.virtualbox] DEBUG: Getting status for cuckoo1
2016-02-27 19:44:58,931 [modules.machinery.virtualbox] DEBUG: Machine cuckoo1 status running
2016-02-27 19:45:00,156 [lib.cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=192.168.56.101)
2016-02-27 19:45:00,557 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for status 0x0001
2016-02-27 19:45:00,570 [lib.cuckoo.core.guest] DEBUG: cuckoo1: status ready
2016-02-27 19:45:00,615 [lib.cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=cuckoo1, ip=192.168.56.101)
2016-02-27 19:45:15,122 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analyzer started with PID 2036
2016-02-27 19:45:15,163 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for completion
2016-02-27 19:45:16,463 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:17,480 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:18,733 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:19,745 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:20,760 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:21,768 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:22,998 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:24,628 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:25,996 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-02-27 19:45:28,078 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)

On the Windows guest machine, the agent keeps getting:

192.168.56.1 - - [27/Feb/2016 16.47.15] "POST /RPC2 HTTP/1.1" 200 -

Can any one tell me what is wrong?

**Comment by jbremer** [comment]
That's just debug logging to indicate that the analysis has not yet finished. It's not actually an error.
Closing this issue as this is intended behavior (to show you that it's still processing the analysis).
If you have any other questions, do let us know ;-)

**Comment by xulifan** [comment]
But after the analysis timeout, it only generates a binary and a dump.pcap in storage/analyses/1, it does not generate anything else. I cannot use process.py to generate the report.

**Comment by Brandl** [comment]
We are running into the same problem, even with timeout set up to 60000.
I would recommend reopening this issue and would happily help investigating.

**Comment by KillerInstinct** [comment]
Usually this is always a networking issue between the host/guest. Can you confirm that they can ping both ways?

**Comment by Brandl** [comment]
Just tested ping both ways, works perfectly.
Also the python shell in the guest shows "200 - POST"-requests, so at least some communication is going on.

**Comment by jbremer** [comment]
Well, what's the actual issue? What I only notice now though is that it would appear that there's no analysis process started in the original comment, so that could be a bug.

**Comment by Brandl** [comment]
The issue is that no dynamic analysis is generated even after 60000 seconds. We tested it with different samples. I appended the log output from the debug server.
[output.txt](https://github.com/cuckoosandbox/cuckoo/files/184085/output.txt)

**Comment by nujozet** [comment]
Exact same issue here. Pings and curls have the right output. I just get stuck in an infinite _"post /rpc2 http 1.1" 200_ until the timeout is reached. Tested with a timeout arround a week (600000 secs).
Cuckoo on CentOS 7 64 bits, guest Windows 7 Home Premium SP1 64 bits.

**Comment by chuddude** [comment]
Hello, same problem here.. 
Is there a solution?

**Comment by blackstone-real** [comment]
i have the exact same issue and was able to reproduce in both UBUNTU 12 and MAC OSX El Capitan. repeated POST with 200 HTTP response until the timeout is reached. Has anyone been able to go beyond that?

**Comment by dperrie20** [comment]
Exact same problem here too, any solution yet?

**Comment by jbremer** [comment]
Most likely the Guest can't reach the Host because of firewall issues. Please try to test that manually.

**Comment by dperrie20** [comment]
No firewalls are active on the guest or host machines, I have tested this on a variety of guest machines (XP, 7, Ubuntu) receiving the same results on all (also tried re-installing and re-configuring all cuckoo files).

**Comment by sganachari** [comment]
Faced the exact issue. 
Solved it in my case by adding resultserverip  & port for each vbox machine in virtualbox.conf.
These are optional values. But only after adding problem is solved in my case.

**Comment by marklech** [comment]
Same here. I would love to hear if there's any progress on this matter.

**Comment by dperrie20** [comment]
I solved it by adding a result server IP address to each virtual box machine in virtualbox.conf, and changing the result server port to 8000 in cuckoo.conf

**Comment by marklech** [comment]
@dperrie20  Din't help in my case

**Comment by dperrie20** [comment]
hm not sure, I have some spare time so if you want to upload your config files I could have a look see if there is anything majorly different that might cause this.

**Comment by marklech** [comment]
I'm debugging this issue , and as it appears to me  there are some bugs in the darwin analyzer implementation. For one thing, it doesn't have an analyis package for .dmg files which is somewhat basic . Moreover, given a file , it has some issues identifying the relevant analysis package and throws an exception for this . Also, you need to make sure that your agent is listening on port 8000, or otherwise change that to your port value in analayzer.py. Once I'll be able to map all problems , I'll give an update.

**Comment by mmsrubar** [comment]
I'm facing the same issue using `657a5166d711135ee6159c8d0abc599d56d294fe` on Ubuntu 16.04 with physical win7 SP1. 

- both machines can ping each other
- firewalls disabled on both machines + UAC 
- running agent.py under Administrator account didn't help

**Comment by lgommans** [comment]
Same problem here. The strange thing is that it worked before, then I restored a snapshot (because automatic snapshot restoring is also broken in Cuckoo, but that's a separate issue) and now it's not working anymore. I checked all settings: firewall off; runs as admin; some communication is going on because there are a few requests... but in the end, "upload failed".

**Comment by John4235** [comment]
I got the exact same problem, when I run a JPG/PDF analyze, I can see the agent opening the picture but it never close it.
The analyze stop when timeout is hit whith an error.
If I take control of my vm launch by cuckoo and I manually close PDF/Picture viewer, cuckoo correctly stop.
Agent on guest seems to not managing to stop the analyze or maybe cuckoo which can't detect end of analyze.
I precise my guest IP is 192.168.56.10, gateway is correctly configured and I can ping my host on 192.168.56.1.
The host can telnet on agent port.

Here is my cuckoo.py -d output:

`2016-12-01 20:12:06,860 [lib.cuckoo.core.scheduler] DEBUG: Processing task #27
2016-12-01 20:12:06,906 [lib.cuckoo.core.scheduler] INFO: Starting analysis of FILE "sample6.png" (task #27, options "route=none,human=0")
2016-12-01 20:12:07,045 [lib.cuckoo.core.scheduler] INFO: Task #27: acquired machine cuckoo1 (label=win)
2016-12-01 20:12:07,071 [modules.auxiliary.sniffer] INFO: Started sniffer with PID 5427 (interface=vboxnet0, host=192.168.56.10, pcap=/opt/cuckoo/storage/analyses/27/dump.pcap)
2016-12-01 20:12:07,072 [lib.cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2016-12-01 20:12:07,118 [modules.machinery.virtualbox] DEBUG: Starting vm win
2016-12-01 20:12:07,123 [modules.machinery.virtualbox] DEBUG: Getting status for win
2016-12-01 20:12:07,324 [modules.machinery.virtualbox] DEBUG: Machine win status poweroff
2016-12-01 20:12:07,401 [modules.machinery.virtualbox] DEBUG: Using snapshot WinXPAgent01 for virtual machine win
2016-12-01 20:12:07,584 [modules.machinery.virtualbox] DEBUG: Getting status for win
2016-12-01 20:12:07,714 [modules.machinery.virtualbox] DEBUG: Machine win status saved
2016-12-01 20:12:10,370 [modules.machinery.virtualbox] DEBUG: Getting status for win
2016-12-01 20:12:10,729 [modules.machinery.virtualbox] DEBUG: Machine win status running
2016-12-01 20:12:10,808 [lib.cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=192.168.56.10)
2016-12-01 20:12:11,815 [lib.cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2016-12-01 20:12:12,820 [lib.cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2016-12-01 20:12:13,815 [lib.cuckoo.core.guest] DEBUG: cuckoo1: not ready yet

2016-12-01 20:12:15,821 [lib.cuckoo.core.guest] DEBUG: cuckoo1: not ready yet
2016-12-01 20:12:15,875 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for status 0x0001
2016-12-01 20:12:15,900 [lib.cuckoo.core.guest] DEBUG: cuckoo1: status ready
2016-12-01 20:12:15,933 [lib.cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=cuckoo1, ip=192.168.56.10, monitor=latest, size=3805883)
2016-12-01 20:12:22,287 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analyzer started with PID 1996
2016-12-01 20:12:22,326 [lib.cuckoo.core.guest] DEBUG: cuckoo1: waiting for completion
2016-12-01 20:12:23,348 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:24,087 [lib.cuckoo.core.resultserver] DEBUG: LogHandler for live analysis.log initialized.
2016-12-01 20:12:24,406 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:25,424 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:26,436 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:27,450 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:28,466 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:29,487 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:30,499 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:31,514 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:32,533 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:33,546 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:34,570 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:35,583 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:36,593 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:37,618 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:38,640 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:39,666 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:40,092 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=204, ppid=420, name=cmd.exe)
2016-12-01 20:12:40,680 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:40,944 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0001.jpg
2016-12-01 20:12:41,017 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 14827
2016-12-01 20:12:41,725 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:42,280 [lib.cuckoo.core.resultserver] DEBUG: New process (pid=756, ppid=204, name=rundll32.exe)
2016-12-01 20:12:42,624 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0002.jpg
2016-12-01 20:12:42,755 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:42,825 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 25124
2016-12-01 20:12:43,766 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:44,134 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0003.jpg
2016-12-01 20:12:44,366 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 26483
2016-12-01 20:12:44,798 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:45,805 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0004.jpg
2016-12-01 20:12:45,870 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 32430
2016-12-01 20:12:45,881 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:46,926 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:47,284 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0005.jpg
2016-12-01 20:12:47,353 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 36420
2016-12-01 20:12:47,947 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:48,522 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0006.jpg
2016-12-01 20:12:48,626 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 69383
2016-12-01 20:12:48,970 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:49,980 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:51,123 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:52,150 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:52,573 [lib.cuckoo.core.resultserver] DEBUG: File upload request for shots/0007.jpg
2016-12-01 20:12:52,685 [lib.cuckoo.core.resultserver] DEBUG: Uploaded file length: 69549
2016-12-01 20:12:53,167 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:54,192 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:55,217 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:56,238 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:57,260 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:58,281 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:12:59,297 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:00,317 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:01,354 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:02,361 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:03,409 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:04,430 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:05,453 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:06,461 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:07,471 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:08,490 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:09,508 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:10,519 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:11,546 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:12,566 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:13,579 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:14,598 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:15,609 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:16,619 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:17,646 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:18,656 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:19,674 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:20,683 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:21,739 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:22,750 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:23,764 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:24,776 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:25,802 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:26,824 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:27,836 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:28,846 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:29,855 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:30,865 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:31,885 [lib.cuckoo.core.guest] DEBUG: cuckoo1: analysis not completed yet (status=2)
2016-12-01 20:13:32,891 [lib.cuckoo.core.scheduler] ERROR: Error from the Cuckoo Guest: The analysis hit the critical timeout, terminating.
2016-12-01 20:13:32,972 [lib.cuckoo.core.plugins] WARNING: Unable to stop auxiliary module: Potential error while running tcpdump, did not expect the following standard error output: 'dropped privs to root'.
2016-12-01 20:13:32,972 [modules.machinery.virtualbox] DEBUG: Stopping vm win
2016-12-01 20:13:32,973 [modules.machinery.virtualbox] DEBUG: Getting status for win
2016-12-01 20:13:33,401 [modules.machinery.virtualbox] DEBUG: Machine win status running
2016-12-01 20:13:34,445 [modules.machinery.virtualbox] DEBUG: Getting status for win
2016-12-01 20:13:34,623 [modules.machinery.virtualbox] DEBUG: Machine win status poweroff
2016-12-01 20:13:37,111 [lib.cuckoo.core.scheduler] DEBUG: Released database task #27
2016-12-01 20:13:37,186 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,189 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,300 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,301 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,305 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Debug" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,306 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,315 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,317 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,324 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Static" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,568 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Strings" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,790 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:37,808 [modules.processing.network] DEBUG: Whitelisting Disabled.
2016-12-01 20:13:37,810 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:39,065 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "VirusTotal" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:39,066 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" on analysis at "/opt/cuckoo/storage/analyses/27"
2016-12-01 20:13:39,105 [lib.cuckoo.core.plugins] DEBUG: Running 418 signatures
2016-12-01 20:13:41,272 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2016-12-01 20:13:41,630 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "MongoDB"
2016-12-01 20:13:41,633 [lib.cuckoo.core.scheduler] INFO: Task #27: reports generation completed (path=/opt/cuckoo/storage/analyses/27)
2016-12-01 20:13:41,687 [lib.cuckoo.core.scheduler] INFO: Task #27: analysis procedure completed
`

**Comment by lovina37** [comment]
Failed to run the reporting module "MongoDB":

**Comment by doomedraven** [comment]
@lovina37 provided better logs

**Comment by emilianooddo** [comment]
I have the same error : `2018-10-13 23:57:16,140 [lib.cuckoo.core.scheduler] ERROR: The analysis hit the critical timeout, terminating.
2018-10-13 23:57:16,225 [modules.machinery.avd] DEBUG: Stopping vm aosx_1
2018-10-13 23:57:16,226 [modules.machinery.avd] INFO: Stopping AVD listening on port 5554
2018-10-13 23:57:16,352 [lib.cuckoo.core.resultserver] DEBUG: Connection closed: 127.0.0.1:56906
2018-10-13 23:57:17,489 [lib.cuckoo.core.scheduler] DEBUG: Released database task #8 with status False
2018-10-13 23:57:17,511 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,659 [modules.processing.apkinfo] WARNING: Dex size bigger than: None
2018-10-13 23:57:17,659 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "ApkInfo" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,660 [modules.processing.behavior] INFO: Analysis results folder does not contain any file or injection was disabled.
2018-10-13 23:57:17,660 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,663 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Debug" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,664 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Droidmon" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,664 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,665 [lib.cuckoo.core.plugins] ERROR: Failed to run the processing module "NetworkAnalysis":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 186, in process
    data = current.run()
  File "/home/emiliano/cuckoo/modules/processing/network.py", line 593, in run
    sort_pcap(self.pcap_path, sorted_path)
  File "/home/emiliano/cuckoo/modules/processing/network.py", line 708, in sort_pcap
    batch_sort(inc, outpath, output_class=lambda path: SortCap(path, linktype=inc.linktype))
  File "/home/emiliano/cuckoo/modules/processing/network.py", line 635, in batch_sort
    current_chunk = list(islice(input_iterator,buffer_size))
  File "/home/emiliano/cuckoo/modules/processing/network.py", line 677, in __iter__
    self.fd = dpkt.pcap.Reader(open(self.name, "rb"))
IOError: [Errno 2] No such file or directory: '/home/emiliano/cuckoo/storage/analyses/8/dump.pcap'
2018-10-13 23:57:17,665 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,667 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Static" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,688 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "Strings" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:17,702 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:18,634 [lib.cuckoo.core.plugins] DEBUG: Executed processing module "VirusTotal" on analysis at "/home/emiliano/cuckoo/storage/analyses/8"
2018-10-13 23:57:18,635 [lib.cuckoo.core.plugins] DEBUG: Applying signature overlays for signatures: creates_exe
2018-10-13 23:57:18,636 [lib.cuckoo.core.plugins] DEBUG: Running non-evented signatures
2018-10-13 23:57:18,636 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_AirPush_Adware"
2018-10-13 23:57:18,638 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_Umeng_Adware"
2018-10-13 23:57:18,638 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_antivirus_virustotal"
2018-10-13 23:57:18,638 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "android_antivirus_virustotal"
2018-10-13 23:57:18,639 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_antivirus_virustotal_threshold"
2018-10-13 23:57:18,639 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "android_antivirus_virustotal_threshold"
2018-10-13 23:57:18,639 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_dangerous_permissions"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "android_dangerous_permissions"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_dynamic_code"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_google_play_diff"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_hidden_payload"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "android_hidden_payload"
2018-10-13 23:57:18,640 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_maware_snadrorat"
2018-10-13 23:57:18,641 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_maware_androrat"
2018-10-13 23:57:18,641 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_maware_iBanking"
2018-10-13 23:57:18,642 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_native_code"
2018-10-13 23:57:18,642 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_packer"
2018-10-13 23:57:18,642 [lib.cuckoo.core.plugins] DEBUG: Running signature "android_reflection_code"
2018-10-13 23:57:18,642 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_aborted_broadcast_receiver"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_contains_apk"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "application_contains_apk"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_contains_arm_binaries"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_contains_dex"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_contains_jar"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_contains_so"
2018-10-13 23:57:18,643 [lib.cuckoo.core.plugins] DEBUG: Analysis matched signature "application_contains_so"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_device_admin"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_dropped_dex"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_dropped_files"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_dropped_so"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_executed_shell_command"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_fingerprint"
2018-10-13 23:57:18,644 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_installed_app"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_queried_account_info"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_queried_installed_apps"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_queried_private_information"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_recording_audio"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_reflection"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_registered_receiver_runtime"
2018-10-13 23:57:18,645 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_root"
2018-10-13 23:57:18,646 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_sent_sms_messages"
2018-10-13 23:57:18,646 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_setComponentEnabledSetting"
2018-10-13 23:57:18,646 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_stopped_processes"
2018-10-13 23:57:18,646 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_uses_location"
2018-10-13 23:57:18,646 [lib.cuckoo.core.plugins] DEBUG: Running signature "application_using_the_camera"
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] DEBUG: Running signature "creates_exe"
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] DEBUG: Running signature "generic_metrics"
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] DEBUG: Running signature "network_http"
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] ERROR: Failed to run signature "network_http":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 342, in process
    if current.run():
  File "/home/emiliano/cuckoo/modules/signatures/network_http.py", line 27, in run
    if "http" in self.results["network"]:
KeyError: 'network'
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] DEBUG: Running signature "network_icmp"
2018-10-13 23:57:18,647 [lib.cuckoo.core.plugins] ERROR: Failed to run signature "network_icmp":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 342, in process
    if current.run():
  File "/home/emiliano/cuckoo/modules/signatures/network_icmp.py", line 27, in run
    if "icmp" in self.results["network"]:
KeyError: 'network'
2018-10-13 23:57:18,648 [lib.cuckoo.core.plugins] DEBUG: Running signature "network_irc"
2018-10-13 23:57:18,648 [lib.cuckoo.core.plugins] ERROR: Failed to run signature "network_irc":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 342, in process
    if current.run():
  File "/home/emiliano/cuckoo/modules/signatures/network_irc.py", line 27, in run
    if "irc" in self.results["network"]:
KeyError: 'network'
2018-10-13 23:57:18,648 [lib.cuckoo.core.plugins] DEBUG: Running signature "network_smtp"
2018-10-13 23:57:18,648 [lib.cuckoo.core.plugins] ERROR: Failed to run signature "network_smtp":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 342, in process
    if current.run():
  File "/home/emiliano/cuckoo/modules/signatures/network_smtp.py", line 27, in run
    if "smtp" in self.results["network"]:
KeyError: 'network'
2018-10-13 23:57:18,688 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"
2018-10-13 23:57:18,818 [lib.cuckoo.core.plugins] ERROR: Failed to run the reporting module "MongoDB":
Traceback (most recent call last):
  File "/home/emiliano/cuckoo/lib/cuckoo/core/plugins.py", line 505, in process
    current.run(self.results)
  File "/home/emiliano/cuckoo/modules/reporting/mongodb.py", line 216, in run
    self.conn.disconnect()
  File "/home/emiliano/.local/lib/python2.7/site-packages/pymongo/database.py", line 1053, in __call__
    self.__name, self.__client.__class__.__name__))
TypeError: 'Database' object is not callable. If you meant to call the 'disconnect' method on a 'MongoClient' object it is failing because no such method exists.
2018-10-13 23:57:19,103 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "ReportAndroidHTML"
2018-10-13 23:57:19,384 [lib.cuckoo.core.plugins] DEBUG: Executed reporting module "ReportHTML"
2018-10-13 23:57:19,384 [lib.cuckoo.core.scheduler] INFO: Task #8: reports generation completed (path=/home/emiliano/cuckoo/storage/analyses/8)
2018-10-13 23:57:19,597 [lib.cuckoo.core.scheduler] INFO: Task #8: analysis procedure completed
`
So, I haven't dynamic analysis, but only static. How can I do? Where is the problem? thanks
